### ISSUE TITLE
[bug]The app is drawing in an untappable area.

### ISSUE THREAD

**Issue by atuy1219** [cylonid/NativeAlphaForAndroid#138]
The app is drawing in an untappable area.
 This is causing the three-dot menu to become inaccessible. This bug also occurs in webviews.
![Screenshot_20240822-201606](https://github.com/user-attachments/assets/b254bf98-df7d-4d74-adb3-f0ad2b66608b)
Android 15 Pixel 6a

**Comment by atuy1219** [comment]
If you use Material 3 components, you might be able to resolve this issue.

**Comment by atuy1219** [comment]
If you use Material 3 components, you might be able to resolve this issue.
![Screenshot_20240822-223227](https://github.com/user-attachments/assets/79033289-fe83-4cf0-9ab0-a3d90870207e)

**Comment by cylonid** [comment]
Thank you for bringing this to my attention, will be fixed with next release.

**Comment by pixincreate** [comment]
Yes, targetSDK 35 by default uses edge to edge UI.

**Comment by domacikolaci** [comment]
Any chance of this one getting fixed? Its rendered the app unusable for me.

**Comment by Aasikki** [comment]
> Any chance of this one getting fixed? Its rendered the app unusable for me.

Yeah this makes it basically unusable on Android 15. Fix would be really appreciated!

**Comment by apparle** [comment]
I'll also appreciate a hot fix / quicker release just for this bug. And I'll be happy to help test this.
With this bug, it has become unusable for me too 😔

**Comment by cylonid** [comment]
Sorry it took so long to fix, the problem was the old adblock engine was no longer available and I did not want to release an update with less functionality then before. 

This issue itself is easy to fix by setting a flag, I am not really fond of this edge-to-edge enforcement and will postpone the change for the moment. 

Everything is looking good now, if everything goes to plan the update should be online in a few days 🙂
